scripts/gufi_config.py

```diff
@@ -67,9 +67,17 @@
 
 import gufi_common
 
-# default configuration file location
+# configuration file location
 DEFAULT_PATH = '/etc/GUFI/config'
 
+# helper function to allow configurable paths to the gufi config file
+def config_path():
+  global DEFAULT_PATH
+  gufi_env_config='GUFI_CONFIG'
+  if gufi_env_config in os.environ:
+    DEFAULT_PATH = os.environ[gufi_env_config]
```

Comment:
why modify the global variable? why not just return the selected one?

Reply:
I thought it best to modify in case it is used in future. I can remove this if you prefer.

Reply:
remove it please

Reply:
removed